src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -89,6 +90,9 @@ class FrameworkBundle extends Bundle
     public function boot()
     {
         ErrorHandler::register(null, false)->throwAt($this->container->getParameter('debug.error_handler.throw_at'), true);
+        if (class_exists(LegacyErrorHandler::class, false)) {
+            LegacyErrorHandler::register(null, false)->throwAt($this->container->getParameter('debug.error_handler.throw_at'), true);
```

Comment:
doesn't this mean that the legacy one wins when both component are installed ?

Reply:
nope, because of the 2nd argument: `$replace = false`